src/pal/src/misc/environ.cpp

```diff
@@ -231,11 +258,11 @@ lpName
        [in] Pointer to a null-terminated string that specifies the
        environment variable whose value is being set. The operating
        system creates the environment variable if it does not exist
-       and lpValue is not NULL.
+       and lpValue is not null.
```

Comment:
It's not a big deal, but I changed `NULL` to `nullptr` in the code, so I felt the comments shouldn't refer to it as `NULL` any longer.